Firestore/CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
-# Unreleased
-- [fixed] Fix App crashed when there are nested data structures inside IN Filter.
-
 # 10.3.0
 - [feature] Add MultiDb support.
+- [fixed] Fix App crashed when there are nested data structures inside IN
+  Filter (#10535).
```

Comment:
Should be the issue number when there is both a PR and issue - #10507

Reply:
@ehsannas I've changed this to the issue number and committed to your PR.

Reply:
Oh! TIL. Thanks!